plugins/nexus-blobstore-s3/src/main/java/org/sonatype/nexus/blobstore/s3/internal/S3BlobStore.java

```diff
@@ -110,6 +110,8 @@
 
   public static final String SIGNERTYPE_KEY = "signertype";
 
+  public static final String FORCE_PATH_STYLE = "forcepathstyle";
```

Comment:
This should be suffixed with `_KEY` to be consistent with the other configuration keys.

Reply:
Done.